airflow/plugins_manager.py

```diff
@@ -49,6 +49,8 @@ class AirflowPlugin(object):
     admin_views = []
     flask_blueprints = []
     menu_links = []
+    role_based_views = []
+    role_based_menu_items = []
```

Comment:
Lets call these `appbuilder_views` etc. The "role" is just the reason we switched to FAB.